services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -5430,14 +5437,17 @@ private Pair<ExecutionStatus, String> getDaVinciPushStatusAndDetails(
     String statusDetail = null;
     String details = "";
     if (completedPartitions > 0) {
-      details += completedPartitions + "/" + partitionCount + " partitions completed in Da Vinci.";
+      details += completedPartitions + "/" + partitionCount + " partitions completed in" + totalInstanceCount
+          + " Da Vinci instances.";
     }
     if (erroredInstance.isPresent()) {
-      details += "Found a failed instance in Da Vinci, it is " + erroredInstance;
+      details += "Found a failed instance in Da Vinci: " + erroredInstance + ". live instances: " + liveInstanceCount
```

Comment:
Shall we log in the controller too, besides responding to VPJ?

Reply:
dont you think it will be too many such logs in controller for all DVC stores?

Reply:
I somehow agree to the point...My feeling (might not apply to all) is that when we debug the push failure, we tend to look at VPJ log first and then ZK and finally go to problematic server host (highly likely some fault happening there) but we rarely go to controller first. If we are going to log this info in VPJ then I feel like it might be good enough.

Reply:
Will DVC call this API, or only VPJ does? How frequent will this be printed indeed matters.

We wanted to check the VPJ logs during the incident, but logs were gone. If we had it in controller, we could find more info.

Reply:
VPJ logs have 30days retains vs 5days for controllers.